catalog/spatial/wfs/spatial-wfs-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/catalog/common/FeatureMetacardType.java

```diff
@@ -237,28 +242,95 @@ private void processXmlSchemaParticle(XmlSchemaParticle particle) {
     }
   }
 
-  private void processSimpleType(XmlSchemaElement xmlSchemaElement) {
-    QName qName = xmlSchemaElement.getSchemaTypeName();
-    String name = xmlSchemaElement.getName();
-    AttributeType<?> attributeType = toBasicType(qName);
+  private void processSimpleContent(
+      final XmlSchemaElement parentElement, final QName simpleContentTypeName) {
+    final QName baseTypeName = getBaseTypeQName(simpleContentTypeName);
+
+    if (baseTypeName != null) {
+      mapSchemaElement(parentElement, baseTypeName);
+    } else {
+      LOGGER.debug("Could not find the base type for simple content: {}", simpleContentTypeName);
+    }
+  }
+
+  private void processSimpleType(final XmlSchemaElement xmlSchemaElement) {
+    final QName baseTypeName =
+        xmlSchemaElement.getSchemaType() != null
```

Comment:
Should this be `xmlSchemaElement.getSchemaType() intanceof XmlSchemaSimpleType`?

Reply:
Actually, I can remove this ternary. `processSimpleType()` is only called in two places, each of which is inside an `if` block that checks that the schema type of the element passed to this function is an `XmlSchemaSimpleType`.